en/resources/tutorials/templating.md

```diff
@@ -31,36 +29,83 @@ You can provide your own Kitura template engine by implementing the `TemplateEng
 
 ---
 
-## Adding a template engine to your Kitura app
+## Setting up
+See [Getting Started](/{{ page.lang }}/starter/gettingstarted.html) for instructions on how to install Kitura's command-line interface.
+
+To use the template engines we first need to setup a Kitura server. Open a terminal and run the following:  
+1. Navigate to Desktop  
+`cd ~/Desktop`
+2. Create a directory  
+`mkdir KituraTemplateExample`
+3. Change to the new directory  
+`cd KituraTemplateExample`
+4. Create the Kitura server  
+`kitura init`
 
-To use a template engine you need to specify a dependency in `Package.swift` for the engine you want to use. You can specify multiple dependencies if you wish to support more than one template engine (e.g. both Mustache and Stencil). In our example we'll add both templates to demonstrate GRMustache and Stencil integration.
+This will take a couple of minutes to complete, `kitura init` downloads Kitura and various supporting packages for monitoring the status of your server and configuration files to allow you to easily deploy to a Docker container, a Kubernetes cluster, or the IBM Cloud. 
 
+## Adding a template engine to your Kitura app
+Kitura is a lightweight, modular framework &mdash; this means that by default you get only basic HTTP functionality, like routing, parameter handling, parsing request body, etc. You can augment Kitura's out-of-the-box functionality by adding additional packages using the [Swift Package Manager](https://swift.org/package-manager/), or [SPM](https://swift.org/package-manager/) for short. The `kitura init` command we ran earlier added some of these packages for you already, we need to add two more. The two packages we'd like to add are the template engines we'd like to use: 
```

Comment:
I would remove
"this means that" and "only"
from the first sentence.
Mention kitura init created the package.swift
"The `kitura init` command we ran earlier created a `Package.swift` file with the packages for a basic Kitura server already. We would like to add two packages for the template engines we're going to use:"